packages/itwinui-react/src/core/DatePicker/DatePicker.tsx

```diff
@@ -190,6 +190,34 @@ type DatePickerProps = {
    * @default false
    */
   showYearSelection?: boolean;
+  /**
+   * Allows props to be passed for calendar month year
+   */
+  calendarProps?: React.ComponentProps<'div'>;
+  /**
+   * Allows props to be passed for calendar month
+   */
+  calendarMonthProps?: React.ComponentProps<'span'>;
+  /**
+   * Allows props to be passed for days of the week wrapper
+   */
+  daysOfWeekProps?: React.ComponentProps<'div'>;
+  /**
+   * Allows props to be passed for calendar weekdays
+   */
+  weekDayProps?: React.ComponentProps<'div'>;
+  /**
+   * Allows props to be passed for calendar days (inside & outside)
+   */
+  dayProps?: React.ComponentProps<'div'>;
+  /**
+   * Allows props to be passed for listbox
+   */
+  listboxProps?: React.ComponentProps<'div'>;
```

Comment:
Maybe we can rename `calendarProps` and `listboxProps` as it does not depict the purpose? Maybe listboxProps can become calendarProps and we can name calendarProps to something else.

Reply:
That is not what I asked for.